pipelines/deployer/deployer.yaml

```diff
@@ -484,8 +484,8 @@ resources:
     organization: ((config-organization))
     repository: ((config-repository))
     github_api_token: ((github-api-token))
-    approvers: [chrisfarms]
-    required_approval_count: 0
+    approvers: ((github-approvers))
```

Comment:
Maybe these should be `((config-approvers))` and `((config-approval-count))` ... since they are now specific to the config repo

Reply:
Okay, done for this PR. I imagine we need to rename it in the cluster config repos now too, and also I still don't know how the approvers stuff gets here.

Reply:
yeah you'll need to add `config-approvers` and `config-approval-count` to the cluster-config repos (ie verify-cluster-config/cluster.yaml and techops-cluster-config/sandbox/cluster.yaml

Reply:
okay, what do I set for approvers though?